src/reanimated2/commonTypes.ts

```diff
@@ -37,11 +37,31 @@ export interface StyleProps extends ViewStyle, TextStyle {
   [key: string]: any;
 }
 
-export interface AnimatedStyle
-  extends Record<string, Animation<AnimationObject>> {
+export interface AnimatedStyle extends Record<string, AnimationObject> {
   [key: string]: any;
-  transform?: Array<Record<string, Animation<AnimationObject>>>;
+  transform?: Array<
```

Comment:
I don't think hardcoding it like this is beneficial for us. I'd rather revert those changes and go for more elegant solution.